src/musicbrainz/web/musicbrainzrecordingstask.cpp

```diff
@@ -118,22 +118,15 @@ bool MusicBrainzRecordingsTask::doStart(
         return false;
     }
 
+    // It is not necessary to connect the QNetworkReply::errorOccurred
+    // signal (since Qt 5.15). Network errors are also received trough
```

Comment:
We are supporting Qt < 5.15 for 2.3 though.

Reply:
This is just a comment to prevent anyone from reconnecting one of the error signals again. The deleted code used a different signal for Qt < 5.15, but that doesn't matter for the comment.

Reply:
If that's your intention, then clarify in the comment why it would be wrong or unnecessary to reconnect that signal.

Reply:
The documentation of QNetworkReply::errorOccurred contains the background information, therefore it makes sense to refer to it here.

> he finished() signal will probably follow, indicating that the connection is over.

Reply:
If I understand the comment as it is written, QNetworkReply::finished does all that is needed here but it only works that way in Qt >= 5.15. Did I misunderstand that?

Reply:
I fell into this trap, someone else may too.

Reply:
Okay, I understand after reading the Qt documentation now. I think the comment would be clearer by simply removing `(since Qt 5.15)`. The way you have written it, I understood the comment as saying the behavior of QNetworkReply::finished changed in Qt 5.15, but actually it is QNetworkReply::errorOccurred that was added in Qt 5.15.

Reply:
This is exactly the point what reviews are for! I thought explicitly mentioning the Qt version could be helpful ;)